static/js/publisher/release/actions/revisions.js

```diff
@@ -0,0 +1,8 @@
+export const UPDATE_REVISIONS = "UPDATE_REVISIONS";
+
+export function updateRevisions(revisions) {
```

Comment:
Oh cool, you are using action creators, just not consistently :)

Reply:
As I mentioned, it's much of a work in progress where I move parts of state/features into redux one by one. So it's slowly getting into good shape.